dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -51,7 +51,7 @@ const TableWithFavorite = () => {
     dispatch(getFavoritedDatasets());
   }, [dispatch]);
 
-  const { publicData, favoriteRepoNames } = useSelector(
+  const { publicData, favoriteRepoNames,tableData } = useSelector(
```

Comment:
Looks like you need to run Prettier again.  🙃

It would be good to develop a habit of running Prettier before each commit.